flytekit/configuration/plugin.py

```diff
@@ -0,0 +1,74 @@
+"""Defines a plugin API allowing other libraries to modify the behavior of flytekit.
+
+Libraries can register by defining an object that follows the same API as FlytekitPlugin
+and providing entry pont with the group name "flytekit.plugin". In `setuptools`,
+you can specific them with:
+
+```python
+setup(entry_points={
+    "flytekit.plugin": ["my_plugin=..."]
```

Comment:
is the `...` meant to be the fully qualified class name?

Reply:
Yea, I updated the `...` with more detail.